pkgs/development/libraries/accounts-qt/default.nix

```diff
@@ -11,8 +11,29 @@ mkDerivation rec {
     owner = "accounts-sso";
```

Comment:
Can we shuffle that into normal ordering?

Reply:
This may end up dropped in the future, and I'd rather do cleanups later.